_studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp

```diff
@@ -2962,6 +2962,28 @@ mfxStatus VideoVPPHW::Close()
 } // mfxStatus VideoVPPHW::Close()
 
 
+bool VideoVPPHW::UseCopyPassThrough(const DdiTask *pTask) const
```

Comment:
the name of the function should probably be: Use**Cm**CopyPassThrough(), right?

Reply:
The plan was to hide mere presence of Cm. Why may a user of CopyPassThrought be interested in what is used inside? One just want fastest version, I think.

Reply:
Because it seems that this function implies that CM is going to be used. At least that's what I think looking into last line in its code.

Reply:
Ok, I got it after offline discussion. Under copy pass through we understand any mem copy operation without VP algorithms. This can be map+memcpy or cmcopy. In different modes we prefer different copy type.